models/group.js

```diff
@@ -0,0 +1,10 @@
+const { connector } = require("./databaseUtil");
+
+const group = {
+    groupName: { type: String, required: true },
+    student_ids: { type: Array, required: true }, //array of number
```

Comment:
I don't thing thats how you define arrays in mongoose. I doubt thatll work as expected in MongoDB. Seeing [Mongoose Schema Types](https://mongoosejs.com/docs/schematypes.html), It would work as expected if you simply do 
`student_ids: { type: [Number], required: true }`
instead.